compose-builder/add-security.yml

```diff
@@ -138,6 +138,7 @@ services:
       - edgex-init:/edgex-init:ro,z
       - vault-config:/vault/config:z
       - /tmp/edgex/secrets:/tmp/edgex/secrets:z
+      - kong:/tmp/kong
```

Comment:
Add `:z` in order to make this shareable across containers in the SELinux-enabled regression test environment.

Reply:
Done